swdc

```diff
@@ -66,13 +66,43 @@ function applyFixture()
   docker-compose run --rm cli mysql -h mysql -u root -proot $SHOPWARE_PROJECT < "${DIR}/fixtures/${FIXTURE_NAME}.sql"
 }
 
+function isComposerProject()
+{
+    if [ -f "$SHOPWARE_FOLDER/app/install.sh" ]; then
```

Comment:
Maybe check only app folder. Cause the install file has been changed for a meanwhile

Reply:
```suggestion
    if [ -d "$SHOPWARE_FOLDER/app" ]; then
```
Like this?

Reply:
Yep